CMakeLists.txt

```diff
@@ -862,7 +861,7 @@ if(OPT_PYTHON AND PYTHON_EXECUTABLE)
   set(OPENRAVEPY_VER_INSTALL_DIR "${OPENRAVEPY_INSTALL_DIR}/${OPENRAVEPY_VER_NAME}") # used by openrave-config.cmake
 
   # build python bindings, either with pybind11 or Boost.Python
-  set(USE_PYBIND11_PYTHON_BINDINGS OFF CACHE BOOL "Build OpenRAVE's python bindings with pybind11 (ON) or Boost.Python (OFF)")
+  set(USE_PYBIND11_PYTHON_BINDINGS ON CACHE BOOL "Build OpenRAVE's python bindings with pybind11 (ON) or Boost.Python (OFF)")

```

Comment:
@tgn3000 for now, can you make the default OFF?

Reply:
Merged from `pybind11_stable` branch by mistake. Set OFF.